tests/mainnet.rs

```diff
@@ -65,8 +65,11 @@ fn mainnet_blocks_from_rpc() {
 
             // TODO: Snapshot with consistent ordering for ease of diffing.
             // Currently PlainAccount's storage ordering isn't consistent.
-            let accounts: BTreeMap<Address, PlainAccount> =
-                rpc_storage.get_cache_accounts().into_iter().collect();
+            let accounts: BTreeMap<Address, PlainAccount> = rpc_storage
+                .get_cache_accounts()
+                .into_iter()
```

Comment:
Now I see why you were motivated to store `EvmAccount` directly on disk in the same PR :pray:. 

Let's do it next!